src/ask-questions.js

```diff
@@ -13,7 +13,7 @@ const utils = require('./utils')
 module.exports = async (projectInfos, useDefaultAnswers) => {
   const questions = flatMap(Object.values(questionsBuilders), questionBuilder =>
     questionBuilder(projectInfos)
-  )
+  ).filter(question => question) // filter for null values
```

Comment:
Why would a question equal null ?

Reply:
Ok I understood by reading the end of the PR. Just use `when` from inquirer for conditionnal questions and you won't have to do this 😉